docs/cypress-testing-library/intro.md

```diff
@@ -45,9 +45,9 @@ cy.get('form').then(subject => {
 ```
 
 `cypress-testing-library` supports both jQuery elements and DOM nodes. This is
-necessary because Cypress uses jQuery elements, while `dom-testing-library`
+necessary because Cypress uses jQuery elements, while `DOM Testing Library`
 expects DOM nodes. When you pass a jQuery element as `container`, it will get
 the first DOM node from the collection and use that as the `container` parameter
-for the `dom-testing-library` functions.
+for the `DOM Testing Library` functions.
```

Comment:
Let's remove the backtics around these. It only _sorta_ made sense before, but I don't think it makes any sense now.

Reply:
Done! 👍 

Yet, backticks are used in several other places ([here](https://github.com/testing-library/testing-library-docs/pull/125/files#diff-4b86fa7efa07bfd9ab506f697f6999daR8), for instance, or a couple of lines above your comment).

Shoud we remove all backticks when talking about the library, now that it doesn't look like "code" anymore? Or there's another reason to modify this particular line I'm failing to see?